src/main/java/pageObjects/Admin_Portal/LoginPage.java

```diff
@@ -0,0 +1,47 @@
+package pageObjects.Admin_Portal;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class LoginPage extends BasePage {
+    public LoginPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public void login(String username, String password) {
+
+        driver.findElement(By.id("username")).sendKeys(username);
```

Comment:
This login methods has all the needed steps but it is not used. Use this method and avoid using enterUsername, enterPassword, clickLoginButton methods